controllers/backup_it_utils_test.go

```diff
@@ -285,3 +289,14 @@ func (ts *BackupTestSuite) notifyObserverOfBackupJobStatusChange(status k8upObse
 	event.Event = status
 	observer.GetUpdateChannel() <- event
 }
+
+func (ts *BackupTestSuite) newBackupWithTags() *k8upv1.Backup {
+	BackupWithTags := ts.newBackup()
+	BackupWithTags.Spec.Tags = []string{backupTag}
+	return BackupWithTags
```

Comment:
variable names should be camelCase
```suggestion
	backupWithTags := ts.newBackup()
	backupWithTags.Spec.Tags = []string{backupTag}
	return backupWithTags
```